### DISCUSSION TITLE
Increasing target for rollup to be ES2015+

### DISCUSSION THREAD

**Discussion by andparsons** [motiondivision/motionone#4] [Ideas]
Howdy,

How do you feel about increasing the target for rollup to be ES2015+ instead of it's current es5 target here:
https://github.com/motiondivision/motion/blob/v6.0.1-alpha.24/tsconfig.json#L3

From what I can see, the support for  WAAPI support matches at least ES2015.

I think the extra bundle size savings would be worth it since it wouldn't reduce existing support.

**Comment by mattgperry** [comment]
Yeah I think this is a good idea, it’s probably about time this was revisited (I haven’t changed the target since Popmotion). Although we do have a graceful instant transition fallback which would break so I’ll have to look into what sort of features this would change

**Reply by mattgperry** [reply]
I've bumped it to es2015 which has provided a nice chunk of savings (vanilla 1.85 -> 1.75, react 3.3 -> 2.8), nice suggestion

**Reply by Andarist** [reply]
> Although we do have a graceful instant transition fallback which would break so I’ll have to look into what sort of features this would change

You can always just document that if somebody wants to target pre-es2015 envs then it's up to them to transpile the code here (which can be easily configured in most tools, the only problem is that the default is often to ignore `node_modules` entirely to speed up builds but this can usually be tweaked).
